resources/config/laravel-fractal.php

```diff
@@ -15,4 +15,6 @@
 
     'default_serializer' => '',
 
+    'base_url' => null,
```

Comment:
Could you add a comment block that explains what this option does and also update the config file example in the readme?